website_cookieconsent/data/website_cookieconsent_data.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<openerp>
+    <data noupdate="1">
+        <record id="cookie_consent_config" model="ir.config_parameter">
+            <field name="display_name">Cookie Consent Config</field>
+            <field name="key">website.cookie_consent_config</field>
+            <field name="value">{}</field>
+        </record>
+        <record id="cookie_iframemanager_config" model="ir.config_parameter">
+            <field name="display_name">Cookie Consent Iframemanager Config</field>
+            <field name="key">website.iframemanager_config</field>
+            <field name="value">{}</field>
```

Comment:
Shouldn't we put some defaults here?

Reply:
Adding defaults leads to xml syntax errors I saw its better to leave it empty and add usage example to the github repo https://github.com/orestbida/iframemanager#how-to-set-attributes-on-the-iframe-element

Reply:
Okay. XML syntax errors I think can be avoided by using `CDATA`